public/img/spectrum.svg

```diff
@@ -0,0 +1,11 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="24" height="24" viewBox="0 0 24 24">
+  <defs>
+    <polygon id="spectrumchat-a" points="0 0 23.641 0 23.641 23.399 0 23.399"/>
+  </defs>
+  <g fill="none" fill-rule="evenodd">
+    <mask id="spectrumchat-b" fill="#fff">
+      <use xlink:href="#spectrumchat-a"/>
```

Comment:
I think we should redo this svg to not use a definition and mask and just have a plain shape with curves instead.